File: frontend/projects/domain/src/lib/retention-impact.ts
export interface RetentionImpact {
  id: string;
  months: number;
  expiresAt: string | null;
  version: number;
  state: string;
  impactRevision: string;
  photoCount: number;
  publishedReferences: number;
  unreviewedRequests: number;
}

File: backend/src/Qbs.Domain/RetentionImpact.cs
namespace Qbs.Domain;

public sealed record RetentionImpact(
    Guid Id,
    int Months,
    DateTimeOffset? ExpiresAt,
    long Version,
    string State,
    string ImpactRevision,
    int PhotoCount,
    int PublishedReferences,
    int UnreviewedRequests
);
